### DISCUSSION TITLE
Ho do load json (new units)

### DISCUSSION THREAD

**Discussion by MohMehrnia** [angularsen/UnitsNet#994] [General]
Ho do load JSON (new units, custom unit) on application startup?

**Comment by angularsen** [comment]
Hi, have you checked this wiki out?

https://github.com/angularsen/UnitsNet/wiki/Extending-with-Custom-Units

**Comment by MohMehrnia** [comment]
yes and i can not found solution for import JSON in run time

**Comment by angularsen** [comment]
Ok now I understand. You are right, there is no support to read JSON at runtime. JSON is only read by the CodeGen application to generate C# code. So you may want to take a look at that one.
